05/app.js

```diff
@@ -1,3 +1,40 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+const para = document.createElement('p')
+const firstArticle = document.createElement('article')
+firstArticle.closest('section')
```

Comment:
Jeśli chcesz skorzystać z tego wyszukania to musisz zapisać gdzieś jego wynik (więćej: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest?retiredLocale=pl)

Przykład: `const section = firstArticle.closest('section')`, pamiętaj tylko, że dopiero co utworzyłeś ten element.